### ISSUE TITLE
is the live site down?

### ISSUE THREAD

**Issue by akhrrbk** [adrianhajdin/project_modern_ui_ux_gpt3#1]
i keep getting a connection resufed error. is the site down?

![image](https://user-images.githubusercontent.com/62600869/150802811-7d1cc3ec-bed1-4999-96ba-57f814aa5e8f.png)

**Comment by Ashutosh102** [comment]
Yep it's down...for me also
![bitmoji](https://sdk.bitmoji.com/render/panel/5b7bff30-d7cc-4141-9f91-d77fa135057f-b93d00e1-1eae-4a41-8a88-f287b6e50de0-v1.png?transparent=1&palette=1&width=246)

**Comment by Code-With-Aagam** [comment]
yes it is also down for me

**Comment by Website-Creations** [comment]
yes appears to be down

**Comment by savin2001** [comment]
It is also down for me too. Maybe @adrianhajdin can link it to a freely hosted domain like Github pages or Netlify

**Comment by adrianhajdin** [comment]
Yes, the live site is down. But everything else is working as expected :)

**Comment by Madkjb** [comment]
site still down but code and files still updated

**Comment by AwesomenessReborn** [comment]
down, browser indicates insecure site.

**Comment by danydin** [comment]
down
